lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -72,6 +72,9 @@ def setup_jdbc_config
     # examples of vendor-specific options can be found in this
     # documentation page: https://github.com/jeremyevans/sequel/blob/master/doc/opening_databases.rdoc
     config :sequel_opts, :validate => :hash, :default => {}
+
+    # Maximum number of times to try connecting to database
+    config :maximum_connection_attempts, :validate => :number, :default => 1
```

Comment:
Can you rename this to `connection_retry_attempts`? Also can you introduce `connection_retry_attempts_wait_time` and default to maybe `500ms`?